testkit/src/main/java/org/apache/calcite/test/QuidemTest.java

```diff
@@ -187,6 +225,41 @@ protected void checkRun(String path) throws Exception {
     }
   }
 
+  private void parseRules(String value) {
+    Pattern pattern = Pattern.compile("([+-])(CoreRules|EnumerableRules)\\.(\\w+)");
+    Matcher matcher = pattern.matcher(value);
+
+    while (matcher.find()) {
+      char operation = matcher.group(1).charAt(0);
+      String ruleSource = matcher.group(2);
+      String ruleName = matcher.group(3);
+
+      try {
+        if (ruleSource.equals("CoreRules")) {
```

Comment:
Maybe this can be the default - if no source is specified, to make the specification of rules shorter